src/main/kotlin/no/nav/syfo/cronjob/VerifyBehandlereForKontorCronjob.kt

```diff
@@ -0,0 +1,75 @@
+package no.nav.syfo.cronjob
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.behandler.BehandlerService
+import no.nav.syfo.behandler.fastlege.FastlegeClient
+import org.slf4j.LoggerFactory
+import java.util.UUID
+
+class VerifyBehandlereForKontorCronjob(
+    val behandlerService: BehandlerService,
+    val fastlegeClient: FastlegeClient,
+) : DialogmeldingCronjob {
+
+    override val initialDelayMinutes: Long = 15
+    override val intervalDelayMinutes: Long = 24 * 60
+
+    override suspend fun run() {
+        verifyBehandlereForKontorJob()
+    }
+
+    suspend fun verifyBehandlereForKontorJob() {
+        val verifyResult = DialogmeldingCronjobResult()
+
+        val behandlerKontorListe = behandlerService.getKontor().filter {
+            it.herId != null && it.dialogmeldingEnabled != null
+        }
+        behandlerKontorListe.forEach { behandlerKontor ->
+            try {
+                val behandlerKontorFraAdresseregisteret = fastlegeClient.behandlereForKontor(
+                    callId = UUID.randomUUID().toString(),
+                    kontorHerId = behandlerKontor.herId!!.toInt()
+                )
+                if (behandlerKontorFraAdresseregisteret != null) {
+                    if (!behandlerKontorFraAdresseregisteret.aktiv) {
+                        // Deaktiver kontor
+                        log.info("VerifyBehandlereForKontorCronjob: Disable dialogmelding for kontor med herId ${behandlerKontor.herId} and partnerId ${behandlerKontor.partnerId}")
+                        behandlerService.disableDialogmeldingerForKontor(behandlerKontor)
+                    } else {
+                        val aktiveBehandlereForKontor = behandlerKontorFraAdresseregisteret.behandlere.filter { it.aktiv }
```

Comment:
Tror det går an å gjøre noe ala `behandlerKontorFraAdresseregisteret.behandlere.partition` for å få to lister med aktive og inaktive (i stedet for å gjøre to filtreringer) :bulb: